internal/telemetry/telemetry.go

```diff
@@ -84,6 +84,8 @@ func (c *client) productChange(namespace Namespace, enabled bool) {
 		products.Products.Profiler = ProductDetails{Enabled: enabled}
 	case NamespaceTracers:
 		// Nothing to do
+	case NamespaceCiVisibility:
+		// Nothing to do
```

Comment:
```suggestion
	case NamespaceTracers, NamespaceCiVisibility:
		// Nothing to do
```

Other than that: is there no CiVis product in telemetry ? 🤔 That's sad

Reply:
yeah I know, we still need to work on that.

Reply:
suggestion applied on [9c8f9e7](https://github.com/DataDog/dd-trace-go/pull/2963/commits/9c8f9e7ce3754968b03f8fa750de3b137371a585)